js/rpg_core/Bitmap.js

```diff
@@ -41,14 +41,67 @@ function Bitmap() {
  *
  */
 
-Bitmap.prototype.initialize = function(width, height) {
-    this._canvas = document.createElement('canvas');
-    this._context = this._canvas.getContext('2d');
-    this._canvas.width = Math.max(width || 0, 1);
-    this._canvas.height = Math.max(height || 0, 1);
-    this._baseTexture = new PIXI.BaseTexture(this._canvas);
-    this._baseTexture.mipmap = false;
-    this._baseTexture.scaleMode = PIXI.SCALE_MODES.NEAREST;
+
+Bitmap.prototype._createCanvas = function(width, height){
+    this.__canvas = document.createElement('canvas');
+    this.__context = this.__canvas.getContext('2d');
+
+    this.__canvas.width = Math.max(width || 0, 1);
+    this.__canvas.height = Math.max(height || 0, 1);
+
+    if(this._image){
+        var w = Math.max(this._image.width || 0, 1);
+        var h = Math.max(this._image.height || 0, 1);
+        this.__canvas.width = w;
+        this.__canvas.height = h;
+
+        this._createBaseTexture(this._canvas);
```

Comment:
I think baseTexture should be always created, not only when `this._image` exists.
If `_baseTexture(get)` is called after `this._image` is set, baseTexture is created by `this._image`. (not `this._canvas`)

1. `_createCanvas` is called (without `this._image`)
1. `this._image` is set
1. `_baseTexture(get)` is called, it causes `_createBaseTexture(this._image)` <- Ohh!!!

It's rare case, but may occur 😖